recipes/ignition-cmake/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import glob
+import os
+
+from conans import CMake, ConanFile, tools
+
+
+class IgnitionCmakeConan(ConanFile):
+    name = "ignition-cmake"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ignitionrobotics/ign-cmake"
+    description = "A set of CMake modules that are used by the C++-based Ignition projects."
+    topics = ("ignition", "robotics", "cmake")
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
Hard to see on mobile but I believe the settings are not used.. can they be removed in this case?

Reply:
I think so. Will remove.

Reply:
Well, the cmake script builds a test library.
I left it in to be certain it would select the correct toolchain.

Reply:
Removing `settings` results in error:
```
ignition-cmake/2.5.0: ERROR: Generator cmake_find_package_multi(file:None) failed
'settings.build_type' doesn't exist
'settings' possible configurations are none
ERROR: 'settings.build_type' doesn't exist
'settings' possible configurations are none
```

Reply:
I think you can remove `cmake_find_package` from `generators`.

Reply:
It's currently not part of `generators`. You mean line:
```python
self.cpp_info.names["cmake_find_package"] = "ignition-cmake{}".format(version_major)
```
?